### PR TITLE
Taboo: Send WAIT-FOR-KICKOFF message

### LINKED ISSUE #21: Taboo/TurnMgr: Notify start of new turn

### PR DESCRIPTION
**Pull request by sbirmi**
The message is only sent when the TurnManager is in KICKOFF_WAIT state.

Closes #21

### CODE REVIEW

File: src/Taboo/TabooRoom.py [hunk 1]
```diff
@@ -95,9 +95,10 @@ def processMsg(self, qmsg):
             return self.__processJoin(qmsg)
 
         # When the last READY message is received, we should call
```
Comment (lambdaFooBar) [inline src/Taboo/TabooRoom.py:97]: I think we can get rid of the comment now that we have the functionality in place.
Or at least make it concise

File: src/Taboo/TurnManager.py [hunk 2]
```diff
@@ -18,8 +22,10 @@
 )
 
 class TurnState(Enum):
+    GAME_START_WAIT = 0
```
Comment (lambdaFooBar) [inline src/Taboo/TurnManager.py:25]: Are GAME_START_WAIT and GAME_OVER really TurnStates?
I can understand the purpose.

Turn when instantiated is in KICKOFF_WAIT, then it goes to RUNNING and then OVER. Isn't this enough.

**Review by lambdaFooBar — APPROVED** [review]

File: src/Taboo/TabooRoom.py (hunk 1 above)
Comment (sbirmi) [reply src/Taboo/TabooRoom.py:97]: the next pull request has a more concise version that I will remove in the future. Marking this as resolved for now.

File: src/Taboo/TurnManager.py (hunk 2 above)
Comment (sbirmi) [reply src/Taboo/TurnManager.py:25]: These are perhaps turn manager states. I am happy to rename the Enum to TurnMgrState if you think that's appropriate.

The distinction between KICKOFF_WAIT and GAME_START_WAIT to make updateState() more robust. We should only send the KICKOFF_WAIT message after all players are in READY state/startNewTurn() has been called the first time.

Your thoughts?

Comment (lambdaFooBar) [reply src/Taboo/TurnManager.py:25]: Yeah, these can be seen as TurnMgr states. 

For a particular Turn instance, if you want to "archive" it, something can be added there.

**Pull request merged**